src/votemanager.cpp

```diff
@@ -380,8 +380,8 @@ CON_COMMAND_CHAT(ve, "Vote to extend the current map.")
 			// there's an extend left after a successfull extend vote
 			g_ExtendState = EExtendState::POST_EXTEND_COOLDOWN;
 
-			// Allow another extend vote after 2 minutes
-			new CTimer(120.0f, false, []()
+			// Allow another extend vote after added time lapses
+			new CTimer(g_iExtendTimeToAdd * 60.0f, false, []()
```

Comment:
C++std is set to C++ 20, should be using smart pointers... Same with the `delete` [here](https://github.com/Source2ZE/CS2Fixes/pull/146/files#diff-2b69d38a4a741a5b1647a74b640f5c033318011b1ab1526ab94d84883a0318a4R96)